changelog.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 ### Changed
+- [Update code to return the correct error message in cases of uninitialised spreadsheets](https://github.com/ballerina-platform/ballerina-standard-library/issues/4505)
```

Comment:
I think we don't need the "Update code to return" phrase in the changelogs. Just  "Updated error message in cases of uninitialized spreadsheets" is better.

@daneshk wdyt?

Reply:
updated in [7cc2d06](https://github.com/ballerina-platform/module-ballerinax-persist.googlesheets/pull/38/commits/7cc2d06b018923a1e05a4f3cd1f8961834b48bcd)